deeplearning4j-modelimport/src/main/java/org/deeplearning4j/nn/modelimport/keras/KerasModel.java

```diff
@@ -539,18 +548,41 @@ protected static Object checkAndGetTrainingField(Map<String,Object> map, String
             throws InvalidKerasConfigurationException {
         /* TODO: how might this break?
          * - mismatch between layer/parameter names?
```

Comment:
Eventually rewrite this to go from layer to weights instead of the other way. For now, put off till after testing.